drivers/ukbus/pci/Makefile.uk

```diff
@@ -0,0 +1,12 @@
+$(eval $(call addlib_s,libukbus_pci,$(CONFIG_LIBUKBUS_PCI)))
+
+CINCLUDES-$(CONFIG_LIBUKBUS_PCI)   += -I$(LIBUKBUS_PCI_BASE)/include
+CXXINCLUDES-$(CONFIG_LIBUKBUS_PCI) += -I$(LIBUKBUS_PCI_BASE)/include
+
+LIBUKBUS_PCI-CINCLUDES-$(CONFIG_LIBUKBUS_PCI) += -I$(LIBUKBUS_PCI_BASE)/include
+
```

Comment:
I think you should also include the platform headers, because `pci_ecam.c` has `#include <uk/bus/platform.h>`.